### ISSUE TITLE
Structural Laminated Glass

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#228]
In the original sketch of the Sun Room: https://github.com/dwyl/home/issues/14#issuecomment-504559126 I used quite a lot of **`glass`** ...
Sadly the price has sky-rocketed since the 2020 pandemic so it's no longer feasible to have as much of it in the build.
But I still want around **8-10 m<sup>2</sup>** of it to allow light through to the lower levels.

http://simulador.giuditta.pt/publicFTP/Tabela_Precos.pdf
![image](https://user-images.githubusercontent.com/194400/230744198-cabc4a90-010c-4648-a380-55d3ca043673.png)

Also came across this calculator used by civil/costing engineers:
http://www.geradordeprecos.info/obra_nova/Fachadas/Vidros/Especiais__seguranca/Vidro_laminado_de_seguranca_0_0_0_1_0_0_0_0_5.html
![image](https://user-images.githubusercontent.com/194400/230744778-28a9fe9c-624c-4685-89c2-386892a9ca37.png)

This is the lower-bound of the cost/price we can have in-mind when finding our eventual supplier.
I'm operating under the assumption of spending 170/m<sup>2</sup> for the glass. 
Though depending on the complexity/quantity needed, it could be marginally cheaper than this.

If it comes to it and the PT-based suppliers want toooo much money, I'm not averse to ordering it from China:
https://www.alibaba.com/trade/search?tab=all&searchText=structural+glass+panels+10mm

e.g: https://www.alibaba.com/product-detail/tempered-laminated-structural-glass-floor-panel_1600795268593.html
![image](https://user-images.githubusercontent.com/194400/230745072-a039d5b5-be59-474f-9864-a987ec109312.png)

Seller: https://vattiglass2.en.alibaba.com/minisiteentrance.html?spm=a2700.details.0.0.76a2733bW9PtSE&from=detail&productId=1600795268593
![image](https://user-images.githubusercontent.com/194400/230745114-8a145829-ee75-483b-8756-38343d31ac8e.png)

https://www.alibaba.com/product-detail/Factory-custom-anti-slip-transparent-interlayer_1600690012663.html?spm=a2700.details.0.0.1c2023a0k1Lyvi
![image](https://user-images.githubusercontent.com/194400/230745327-50c82a21-8c46-499d-8f3f-8b2a64c296a1.png)

@iteles what are your thoughts on _ordering_ a _sample_ from China ASAP so that we can determine how _much_ of it we can use in our _final_ design for the Sun Room? 💭

**Comment by nelsonic** [comment]
Sent the follow-up. ✉️ ✅ https://message.alibaba.com/message/messenger.htm gotta love this URL. 😜

**Comment by nelsonic** [comment]
@iteles if you have time today please let me know your thoughts on the level of clarity in the glass. 
Should we have clear to maximise the light entering the building?
Or "frosted" for privacy/modesty? 💭

**Comment by iteles** [comment]
I don't see clear as an option if we want women or anyone in a skirt or looser shorts to use the 1st floor unfortunately 🙃

**Comment by nelsonic** [comment]
Yeah, agreed. I had already axsked the supplier for the various levels of frosting 🧁 …

**Comment by nelsonic** [comment]
@iteles supplier sent us the following images as requested: 

![image](https://user-images.githubusercontent.com/194400/235538382-a5d782d2-5916-482b-a26f-e928cc4ecfe4.png)

![image](https://user-images.githubusercontent.com/194400/235539113-e077b9b4-60db-41c9-a077-debbda2f981d.png)


![image](https://user-images.githubusercontent.com/194400/235538688-b3f95b27-531d-4efa-95b1-e94c54e98b9e.png)

I'd say these offer decent privacy while still letting in enough light. 🙈 ☀️  
I'm just trying to decide how much to order ... 💭 
A 1 x 1 m "sample" is **`80kg`** and will cost like €100+ to ship here. 💸 
But if we order more it will be a similar shipping cost but take 6 - 8 weeks to arrive. ⏳ 

Thoughts?

**Comment by nelsonic** [comment]
@iteles my thoughts on the glass is pretty simple: I don't think _any_ amount of it will go to "waste". I will use any amount for "projects" e.g. table top

I think if we sat down together and focussed on the Sun Room design for **`T25m`** we could put together the `Glass` requirements and get this ordered. 💭

**Comment by iteles** [comment]
If you are comfortable with the seller and confident the quality of the glass will be good, let's book a 'date' to get this done once the kids are in bed tonight.

**Comment by nelsonic** [comment]
Calendar invite sent: 

<img width="448" alt="image" src="https://user-images.githubusercontent.com/194400/235694366-884294ea-5b36-4ad8-9245-d88d896fdb29.png">

**Comment by nelsonic** [comment]
@iteles still up for doing this pairing session? 💭 
(our evening of computer work got scuppered by 👶 ...)

**Comment by nelsonic** [comment]
This site has lots of good examples/images: https://innovativebuildingproducts.com/glasswalk
![image](https://github.com/dwyl/home/assets/194400/a4b6a6ca-8888-4908-a9be-57c558ef27fd)

![image](https://github.com/dwyl/home/assets/194400/7edc3a3a-348a-4cb0-ab25-920da5b0a76e)

![image](https://github.com/dwyl/home/assets/194400/cff96d3c-385e-4250-9297-78c640f7ca94)
